include/ensmallen_bits/spalera_sgd/spalera_stepsize.hpp

```diff
@@ -172,7 +172,9 @@ class SPALeRAStepsize
         // Faster.
         learningRates /= 2;
 
-        if (arma::any(arma::vectorise(learningRates) <= 1e-15))
+        //if (any(vectorise(learningRates) <= 1e-15))
+        /* if (min(vectorise(learningRates)) <= 1e-15) */
+        if (learningRates.min() <= 1e-15)
```

Comment:
```suggestion
        if (learningRates.min() <= 1e-15)
```
I think this is an improvement overall anyway, nice.